Makefile

```diff
@@ -912,12 +913,29 @@ CMAKE_SETUP_RELEASE_GCC := $(CMAKE_DIR_RELEASE_GCC)/build.ninja
 $(CMAKE_SETUP_RELEASE_GCC):
 	@$(MAKE) cmake_check
 	@mkdir -p $(CMAKE_DIR_RELEASE_GCC)
-	cd $(CMAKE_DIR_RELEASE_GCC); CC='$(GCC)' CXX='$(GPP)' cmake -GNinja $(SRC_DIR) -DCMAKE_BUILD_TYPE=Release $(CMAKE_VALGRIND)
+	cd $(CMAKE_DIR_RELEASE_GCC); CC='$(GCC)' CXX='$(GPP)' cmake -GNinja $(SRC_DIR) -DCMAKE_BUILD_TYPE=Release $(CMAKE_VALGRIND) -DENABLE_PGO=ON -DPROFILE_STATE=use
 .PHONY: pyston_release_gcc
-pyston_release_gcc: $(CMAKE_SETUP_RELEASE_GCC)
+pyston_release_gcc: $(CMAKE_SETUP_RELEASE_GCC) $(CMAKE_DIR_RELEASE_GCC)/.trained
```

Comment:
Personally I'd prefer to have two new build configurations rather than reusing the existing one -- and it's bikeshedding but I would have guessed that the "pgo" binary would be the one that was pgo-optimized :P  ie I would prefer adding a "pyston_release_gcc_pgo" and "pyston_release_gcc_pgo_instrumented"